utilities/code_checks/commit-msg

```diff
@@ -142,8 +149,11 @@ validate_commit_message() {
   # reset warnings
   WARNINGS=()
 
-  # capture the subject, and remove the 'squash! ' prefix if present
+  # capture the subject, and remove special prefixes
   COMMIT_SUBJECT=${COMMIT_MSG_LINES[0]/#squash! /}
+  COMMIT_SUBJECT=${COMMIT_SUBJECT/#fixup! /}
+  COMMIT_SUBJECT=${COMMIT_SUBJECT/#amend! /}
+  COMMIT_SUBJECT=${COMMIT_SUBJECT/#reword! /}
```

Comment:
That's what lead me to the change in the first place. The `commit-msg` check now works for these special commits for autosquashing.